backend/src/main/java/com/empire/World.java

```diff
@@ -193,6 +194,7 @@ public static World startNew(String gmPasswordHash, String obsPasswordHash, Map<
 				double flexFactor = remaining == 1 ? remainingShare : Math.min(remainingShare / 2, Math.random() * .4 + .8);
 				remainingShare -= flexFactor;
 				w.regions.get(r).population = population * flexFactor / nation.coreRegions.size();
+				w.regions.get(r).crops = w.regions.get(r).population * Constants.setupCropsPerCitizen;
```

Comment:
Is there a way to use the Region's plant() method here (probably not as written), or is it intentional to separate setup/initialization from mid-game planting?  Perhaps just make the default value for the Region class be the setup amount? (And if so, this might be a good idea throughout the codebase)

Reply:
Long-term, I want to have a few different game-start scenarios, so for now I want to avoid tying details of the initial conditions of the world into the classes. The "World Factory" doesn't actually have to live in World - or it could be separated in a similar way to World.Advancer, but I don't want to scatter these details into Region, etc.

Not sure I fully follow the second idea - I think you're suggesting have population in Region's constructor and set crops based on that?